vars/runCleanupResource.groovy

```diff
@@ -2,6 +2,10 @@
 
 def call(Map params, String region){
     def current_region = initAwsRegionParam(params.region, region)
+    def current_gce_datacenter = ""
+    if (params.backend == "gce") {
```

Comment:
why this is needed, i.e. what's wrong with passing down params.gce_datacenter ? for supporting random ? 

regardless if `initAwsRegionParam` isn't something specific to AWS it should be renamed.